views/application.pug

```diff
@@ -1,10 +1,17 @@
 html
   head
     link(rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.5/css/bootstrap.min.css" integrity="sha384-AysaV+vQoT3kOAXZkl02PThvDr8HYKPZhNT5h/CXfBThSRXQ6jW5DO2ekP5ViFdi" crossorigin="anonymous")
-
+    link(rel="stylesheet" href="/css/application.css")
   body
     nav(class="navbar navbar-light bg-faded")
       a(class="navbar-brand" href="/") User Information Application
+
+      form(class="in-application-search form-inline float-xs-left")
+        input(class="form-control" type="text" placeholder="Search")
+        ul.search-results
```

Comment:
because this is where we will display the search results we styled this ul and the li's inside it so when we add the <li> tags in the custom.js it adds them to the ul and since they(lis) are already styled, they will look pretty.